python/mrchem/CUBEparser.py

```diff
@@ -29,42 +29,51 @@
 from .input_parser.plumbing import pyparsing as pp
 
 global pc
+global world_unit
+
+def make_cube_vector(file_key, file_val, vector_dir):
+    data_type = "_".join(file_key.split("_")[2:])
+    path_list = sort_paths(file_val)
+    cube_list = []
+    
+    if not os.path.isdir(vector_dir):
+        os.mkdir(vector_dir)
```

Comment:
The standard `Pathlib` module should be used instead of string manipulations and `os` to handle interactions with the filesystem. https://docs.python.org/3/library/pathlib.html For example:

```suggestion
    if not Path(vector_dir).is_dir():
        Path(vector_dir).mkdir()
```

Reply:
I have replaced all `os` uses with path.